docs/website/docs/user-guides/quickstart/quickstart.md

```diff
@@ -13,3 +13,36 @@ You have the option of choosing from the following frameworks for the quickstart
 * [Go](go)
 
 A full list of example applications can be viewed with the `odo registry` command.
+<details>
+<summary>Example</summary>
+
+```shell
+$ odo registry
+? Help odo improve by allowing it to collect usage data. Read about our privacy statement: https://developers.redhat.com/article/tool-data-c? Help odo improve by allowing it to collect usage data. Read about our privacy statement: https://developers.redhat.com/article/tool-data-collection. You can change your preference later by changing the ConsentTelemetry preference. Yes
```

Comment:
I would also remove this line. If this is not the first `odo` command that the user ever executed, they won't see this message.

Otherwise, this looks good.

Reply:
Yeah, thanks for pointing this out. I have removed this line, Please take another pass. Thanks,